src/general/StageHUDBase.cs

```diff
@@ -711,7 +711,7 @@ public void UpdateEnvironmentalBars(BiomeConditions biome)
         var oxygenPercentage = biome.Compounds[oxygen].Ambient * 100;
         var co2Percentage = biome.Compounds[carbondioxide].Ambient * 100;
         var nitrogenPercentage = biome.Compounds[nitrogen].Ambient * 100;
-        var sunlightPercentage = biome.Compounds[sunlight].Ambient * 100;
+        var sunlightPercentage = Math.Round(biome.Sunlight!.Current * 100, 0);
```

Comment:
This line seems to indicate that the assumption is that sunlight must always exist, so someone should really consider if the nullability is right on that field in the conditions.